package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "hello-rails-react",
+  "packageManager": "yarn@3.7.0",
+  "dependencies": {
+    "@hotwired/stimulus": "^3.2.2",
+    "@reduxjs/toolkit": "^2.0.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^9.1.0",
+    "react-router-dom": "^6.21.2",
+    "redux": "^5.0.1",
+    "redux-thunk": "^3.1.0"
+  },
```

Comment:
- Good job done so far, but your `package.json` file is missing some needed scripts when trying to precompile your projects assets (CSS & JS). PLease add the below script to your file.
```Javascript
      "scripts": {
    "build": "webpack --config webpack.config.js"
  },
```